Common/src/geometry/CPhysicalGeometry.cpp

```diff
@@ -5192,46 +5178,15 @@ unsigned short iMarker, jMarker, iMarkerTP, iSpan, jSpan, kSpan = 0;
           jSpan++;
         }
       }
+      if (jSpan != nSpan) SU2_MPI::Error("Panic!",CURRENT_FUNCTION);
 
-      delete [] MyTotValueSpan; delete [] MyValueSpan; delete [] My_nSpan_loc;
-
-#endif
-
-      // check if the value are gathered correctly
-      //
-      //  for (iSpan = 0; iSpan < nSpan; iSpan++){
-      //    if(rank == MASTER_NODE){
-      //      cout << setprecision(16)<<  iSpan +1 << " with a value of " <<valueSpan[iSpan]<< " at flag " << marker_flag <<endl;
-      //    }
-      //  }
+      /*--- Terrible stuff to do but so is this entire bloody function goodness me... ---*/
+      SpanWiseValue[marker_flag -1] = valueSpan;
 
+      sort(SpanWiseValue[marker_flag -1], SpanWiseValue[marker_flag -1]+nSpan);
```

Comment:
👍

Reply:
I should hope they fire anyone who implements sort "inline" especially the quadratic complexity variant.
I also hope the comment was correct because I did not try to understand if this "sort" had something different about it.